pkg/network/common/egressip_test.go

```diff
@@ -1068,10 +1068,10 @@ func TestEgressCIDRAllocation(t *testing.T) {
 		t.Fatalf("%v", err)
 	}
 
-	// You can't mix multiple-egress-IP HA with auto-allocated-egress-IP HA
+	// You can mix multiple-egress-IP HA with auto-allocated-egress-IP HA
 	updateNetNamespaceEgress(eit, &networkv1.NetNamespace{
 		NetID:     45,
-		EgressIPs: []networkv1.NetNamespaceEgressIP{"172.17.0.102", "172.17.1.102"},
+		EgressIPs: []networkv1.NetNamespaceEgressIP{"172.17.0.102", "172.17.1.102"}, // 172.17.0.102 is already allocated above
```

Comment:
comment might be more useful down below with the `assertChanges` output?

Reply:
I am not sure it is, I did it like this to align it with all the other comments in the test, see: 
https://github.com/openshift/sdn/blob/b24f70e994471849788104c52a53f47dd8f511ba/pkg/network/common/egressip_test.go#L1034
or 
https://github.com/openshift/sdn/blob/b24f70e994471849788104c52a53f47dd8f511ba/pkg/network/common/egressip_test.go#L1001